common/datavalues/src/columns/string/transform.rs

```diff
@@ -27,7 +27,7 @@ impl StringColumn {
     where
         F: FnMut(&[u8], &mut [u8]) -> Result<usize>,
     {
-        let mut values: Vec<u8> = Vec::with_capacity(estimate_bytes);
+        let mut values: Vec<u8> = vec![0u8; estimate_bytes];
```

Comment:
I found many places use `std::slice::from_raw_parts_mut`.

Do I need reset it, and then other pr will fixed all togathger? @sundy-li

Reply:
> Do I need reset it, and then other pr will fixed all together?

No need, we ensure the memory is initialized manually. And the old function will be removed after we have functions-v2.